recipes/categorical_anomaly.py

```diff
@@ -123,6 +139,26 @@ def tune_params(self,df, metrics_column):
         print("\nTuned LOF Parameters : {}".format(self.tuned_params))
         return(self.tuned_params)
 
+    def raster_to_vector(self,):
```

Comment:
if there's no non-object arguments, remove the trailing `,` as `def raster_to_vector(self):`

on the other hand, this is a great place to apply method-level flexibility so that one can either initialize the object with a `filepath`, or apply one when running the method -- since conceptually there's no other coupling of the geotif file than this particular method 

```python
def raster_to_vector(self, geotif_path):
    fpath = geotif_path or self.geotif_path
```

also notice the suggestion to rename `self.filepath` to `self.geotif_path` (or something along that line) to better represent its intent than an overly generic term

Reply:
I was wondering about that, I initially had it as just a variable within the function you called, then I thought maybe the self variable would be better? 
But without it being used elsewhere, and as something else the user would have to remember upon calling the class, you're probably right, just do it within (and more specific for the name! :) )